### ISSUE TITLE
cannot reset a staffs auth

### ISSUE THREAD

**Issue by PhanaticD** [Ichbinjoe/MCAuthenticator#30]
a staff member got a new phone, and I cannot reset their auth, it says they are already in reset mode, but they dont get a new map or link, it just asks for their code

**Comment by Ichbinjoe** [comment]
Yes, as I look now this appears to be a flaw in logic. Perform /2fa reset <player> when the player is offline, then see if their 2fa will allow them to re-engage once they log back on.

**Comment by PhanaticD** [comment]
still not fixed?
